android/build.gradle

```diff
@@ -95,6 +96,31 @@ subprojects {
     boolean isSample = project.path.startsWith(":demos:") || project.path.startsWith(":tutorials:")
     boolean usesErrorProne = project.name in useErrorProneProjects
 
+    if (isAndroidLibrary) {
```

Comment:
we would want to run this for java libraries, as well, since we have some rib modules that are non-android like rib-base